addOns/ascanrulesAlpha/src/main/java/org/zaproxy/zap/extension/ascanrulesAlpha/Log4ShellScanRule.java

```diff
@@ -128,18 +129,22 @@ public void scan(HttpMessage msg, String param, String value) {
             ExtensionOast extOast =
                     Control.getSingleton().getExtensionLoader().getExtension(ExtensionOast.class);
             for (String attackPattern : ATTACK_PATTERNS) {
-                HttpMessage testMsg = getNewMsg();
-                Alert alert =
-                        newAlert()
-                                .setConfidence(Alert.CONFIDENCE_MEDIUM)
-                                .setParam(param)
-                                .setMessage(testMsg)
-                                .build();
-                String payload = extOast.registerAlertAndGetPayload(alert);
-                String attack = attackPattern.replace("{0}", payload);
-                alert.setAttack(attack);
-                setParameter(testMsg, param, attack);
-                sendAndReceive(testMsg);
+                try {
```

Comment:
This should be reverted, it's already being done in the other PR.

Reply:
yes, but without that change it can not be tested well.

Should I revert anyhow?

Reply:
I think so, the other should be merged soon.